src/admin/components/elements/Popup/index.tsx

```diff
@@ -30,26 +28,29 @@ const Popup: React.FC<Props> = (props) => {
     boundingRef,
   } = props;
 
+  const { width: windowWidth, height: windowHeight } = useWindowInfo();
+  const [intersectionRef, intersectionEntry] = useIntersect({
+    threshold: 1,
```

Comment:
request review and payment credits issued @ shednandez

0xf752EC3A9091Aa800561f5e232e7b0b326402215

Reply:
0xf752EC3A9091Aa800561f5e232e7b0b326402215